core/src/workflow/machines/mod.rs

```diff
@@ -37,6 +37,7 @@ use patch_state_machine::PatchMachine;
 use prost::alloc::fmt::Formatter;
 use rustfsm::{MachineError, StateMachine};
 use signal_external_state_machine::SignalExternalMachine;
+use upsert_search_attributes_state_machine::UpsertSearchAttributesMachine;
```

Comment:
I can't comment on unchanged lines, but, you'll want to remove the `#[allow(unused)]` attribute at the top of the file for the upsert module

Reply:
Not sure I follow - what is unused?

Reply:
Line 18 has that, you can remove it now

Reply:
Totally misread your initial comment, sorry. I see what you're saying.